e2e/resources/workspace-data.ts

```diff
@@ -0,0 +1,171 @@
+import {makeString} from 'resources/fake-user';
+import {field} from '../app/workspace-edit-page';
```

Comment:
can we use this as 'app/workspace-edit-page'?

Reply:
Done.